picard/ui/scripteditor.py

```diff
@@ -449,43 +598,91 @@ def toggle_wordwrap(self):
             self.ui.file_naming_format.setLineWrapMode(QtWidgets.QTextEdit.NoWrap)
 
     def import_script(self):
-        """Import the current script from an external text file.
+        """Import from an external text file to a new script. Import can be either a plain text script or
+        a naming script package.
         """
+        error_import = 'Error importing "%s". %s.'
+        error_import_tx = N_('Error importing "%s". %s.')
+        error_decode = 'Error decoding "%s". %s.'
+        error_decode_tx = N_('Error decoding "%s". %s.')
+
         dialog_title = N_("Import Script File")
-        dialog_file_types = self.FILE_TYPE_SCRIPT + ";;" + self.FILE_TYPE_ALL
+        dialog_file_types = self.FILE_TYPE_PACKAGE + ";;" + self.FILE_TYPE_SCRIPT + ";;" + self.FILE_TYPE_ALL
         options = QtWidgets.QFileDialog.Options()
         options |= QtWidgets.QFileDialog.DontUseNativeDialog
         filename, file_type = QtWidgets.QFileDialog.getOpenFileName(self, dialog_title, self.default_script_directory, dialog_file_types, options=options)
         if filename:
             log.debug('Importing naming script file: %s' % filename)
             try:
                 with open(filename, 'r', encoding='utf8') as i_file:
-                    self.set_script(i_file.read())
+                    file_content = i_file.read()
             except OSError as error:
-                error_message = 'Error importing file "{0}": {1}'.format(filename, error.strerror)
-                log.error(error_message)
-                self.display_error(OptionsCheckError(N_("File Error"), error_message))
+                log.error(error_import % (filename, error.strerror))
+                error_message = error_import_tx % (filename, error.strerror)
```

Comment:
I thought the standard was to log in English, but display translated pop-ups to the user?  I can change so that these are all translated (there should be 5 or 6 cases like this), if that's preferred.

Reply:
> I thought the standard was to log in English, but display translated pop-ups to the user?

It is, you are missing the point here.
 `N_()` doesn't translate anything, so as is, both are untranslated.
 But the string is the same in both cases, read again the code I suggested (`_(error_import) `)

Reply:
Fixed in https://github.com/metabrainz/picard/pull/1810/commits/e2b3020e982c77b05b63050475bb21d8e24547a8